packages/core/src/constants.ts

```diff
@@ -44,32 +44,27 @@ export type ChainStepTextResponse = {
   text: string
   usage: CompletionTokenUsage
   toolCalls: ToolCall[]
-  documentLogUuid: string
-  providerLog: undefined
+  documentLogUuid?: string
```

Comment:
these can always be optional since not all provider calls are made in the context of a documentloguuid